ops/demo/main.tf

```diff
@@ -90,7 +90,7 @@ module "app_gateway" {
   resource_group_location = data.azurerm_resource_group.rg.location
   resource_group_name     = data.azurerm_resource_group.rg.name
 
-  blob_endpoint     = azurerm_cdn_endpoint.cdn_endpoint.host_name
+  blob_endpoint     = azurerm_cdn_endpoint.cdn_endpoint.fqdn
```

Comment:
fqdn is not the same as host_name, is this an intentional change?

Reply:
You're correct! This was intentional; the azure provider team made the update to fqdn from host_name going into version 3.